packages/scaffold-config/src/frameworks.ts

```diff
@@ -43,7 +43,7 @@ export async function isDependencyInstalled (dependency: Cypress.CypressComponen
     // for module resolution.
     const resolvePackagePath = require('resolve-package-path')
 
-    const packageFilePath = resolvePackagePath(dependency.package, projectPath)
+    const packageFilePath = resolvePackagePath(dependency.package, projectPath, false)
```

Comment:
This is the fix, the third arg is whether to cache or not - we don't want caching, or it won't find the dependency, even if the user adds it.